staging/src/k8s.io/mount-utils/mount_linux_test.go

```diff
@@ -715,19 +699,110 @@ func TestFormat(t *testing.T) {
 			}
 			mounter := NewSafeFormatAndMount(nil, exec, WithMaxConcurrentFormat(tc.max, tc.timeout))
 
-			var wg sync.WaitGroup
+			for i := 0; i < tc.max+1; i++ {
+				go func() {
+					mounter.format(fstype, nil)
+				}()
+			}
+
+			time.Sleep(100 * time.Millisecond)
```

Comment:
comment
// wait for all goorutines to be shceduled

Reply:
resolved in 0a21ba0476